src/components/CheckboxList.tsx

```diff
@@ -55,7 +54,7 @@ const CheckboxesWrapper = styled.div`
   &:not(:last-child) {
     margin-bottom: 12px;
     padding-bottom: 16px;
-    border-bottom: 1px solid ${addOpacityToHsla(blueGrey700, 0.25)};
+    border-bottom: 1px solid ${addOpacityToColor("var(--blue-grey-700)", 0.25)};
```

Comment:
I've updated this function to use the new css `color-mix` property which means it works on any css color string including variables as seen here